content/en/blog/_posts/2023-08-15-pv-last-phase-transtition-time.md

```diff
@@ -0,0 +1,87 @@
+---
+layout: blog
+title: "Kubernetes 1.28: Persistent Volume Last Phase Transition Time"
+date: 2023-08-15T10:00:00-08:00
+slug: kubernetes-1-28-persistent-volume-last-phase-transition-time
+---
+
+**Author:** Roman Bednář (Red Hat)
+
+In Kubernetes v1.28, we are introducing a new alpha feature that aims to improve PersistentVolume (PV) management and help cluster administrators gain better insights into the lifecycle of PVs.
+With the addition of the LastPhaseTransitionTime field in the PV status, administrators will be able to track the last time a PV transitioned to a different phase, allowing for more efficient and informed resource management.
```

Comment:
Please use user facing field names when possible.
This blog is for users, not for developers.
A user doesn't need to know that the field is named as `LastPhaseTransistionTime` in Go lang and the first character must be upper case for it to be treated public.
Instead, he or she need to know the field name as `lastPhaseTransistionTime`.

Reply:
Acknowledged, I've applied your suggestions and replaced all other references to that field to be lower case. Thank you for review.